example/CMakeLists.txt

```diff
@@ -31,6 +31,6 @@ add_executable(osrm-example example.cpp)
 
 find_package(LibOSRM REQUIRED)
 
-target_link_libraries(osrm-example ${LibOSRM_LIBRARIES} ${LibOSRM_DEPENDENT_LIBRARIES})
+target_link_libraries(osrm-example ${LibOSRM_LIBRARIES} ${LibOSRM_DEPENDENT_LIBRARIES} ${UtilGlob})
```

Comment:
Why does the example now need the util lib? Shouldn't all symbols needed for linking against osrm be included in the `libosrm` library already?